datasets/mnist_usps_pca.py

```diff
@@ -0,0 +1,78 @@
+from benchopt import BaseDataset, safe_import_context
```

Comment:
is `mnist_usps_pca.py` the old `mnist_usps.py` ? for next time, you can use `git mv` so that a renaming does not appear as a new file

Reply:
yes, nice to know. Thanks